freemocap/core_processes/process_motion_capture_videos/process_recording_folder.py

```diff
@@ -81,37 +84,48 @@ def process_recording_folder(
-        save_copy_of_calibration_to_this_path=rec.recording_info_model.path,
-    )
-    (raw_skel3d_frame_marker_xyz, skeleton_reprojection_error_fr_mar,) = triangulate_3d_data(
-        anipose_calibration_object=anipose_calibration_object,
-        mediapipe_2d_data=mediapipe_2d_data,
-        output_data_folder_path=rec.recording_info_model.raw_data_folder_path,
-        mediapipe_confidence_cutoff_threshold=rec.anipose_triangulate_3d_parameters_model.confidence_threshold_cutoff,
-        use_triangulate_ransac=rec.anipose_triangulate_3d_parameters_model.use_triangulate_ransac_method,
-        kill_event=kill_event,
-    )
+    else:
 
-    if kill_event is not None and kill_event.is_set():
-        return
+        logger.info("Triangulating 3d skeletons...")
 
-    assert test_mediapipe_3d_data_shape(
-        synchronized_videos_folder=rec.recording_info_model.synchronized_videos_folder_path,
-        mediapipe_3d_data_npy_path=rec.recording_info_model.raw_mediapipe_3d_data_npy_file_path,
-        medipipe_reprojection_error_data_npy_path=rec.recording_info_model.mediapipe_reprojection_error_data_npy_file_path,
-    )
+        assert rec.recording_info_model.calibration_toml_check, f"No calibration file found at: {rec.recording_info_model.calibration_toml_path}"
+        assert rec.recording_info_model.data2d_status_check, f"No mediapipe 2d data found at: {rec.recording_info_model.mediapipe_2d_data_npy_file_path}"
+
+        anipose_calibration_object = load_anipose_calibration_toml_from_path(
+            camera_calibration_data_toml_path=rec.recording_info_model.calibration_toml_path,
+            save_copy_of_calibration_to_this_path=rec.recording_info_model.path,
+        )
+        (raw_skel3d_frame_marker_xyz, skeleton_reprojection_error_fr_mar,) = triangulate_3d_data(
+            anipose_calibration_object=anipose_calibration_object,
+            mediapipe_2d_data=mediapipe_image_data_numCams_numFrames_numTrackedPts_XYZ,
+            output_data_folder_path=rec.recording_info_model.raw_data_folder_path,
+            mediapipe_confidence_cutoff_threshold=rec.anipose_triangulate_3d_parameters_model.confidence_threshold_cutoff,
+            use_triangulate_ransac=rec.anipose_triangulate_3d_parameters_model.use_triangulate_ransac_method,
+            kill_event=kill_event,
+        )
+
+        if kill_event is not None and kill_event.is_set():
+            return
+
+        assert test_mediapipe_3d_data_shape(
+            synchronized_videos_folder=rec.recording_info_model.synchronized_videos_folder_path,
+            mediapipe_3d_data_npy_path=rec.recording_info_model.raw_mediapipe_3d_data_npy_file_path,
+            medipipe_reprojection_error_data_npy_path=rec.recording_info_model.mediapipe_reprojection_error_data_npy_file_path,
+        )
+
+    rotated_raw_skel3d_frame_marker_xyz = rotate_by_90_degrees_around_x_axis(raw_skel3d_frame_marker_xyz)
```

Comment:
```suggestion
    #rotate so skeleton is closer to 'vertical' in a z-up reference frame
    rotated_raw_skel3d_frame_marker_xyz = rotate_by_90_degrees_around_x_axis(raw_skel3d_frame_marker_xyz) 
```

I think this is worth a comment